auth-api/src/auth_api/services/membership.py

```diff
@@ -168,11 +166,17 @@ def update_membership(self, updated_fields, token_info: Dict = None):
         check_auth(org_id=self._model.org_id, token_info=token_info, one_of_roles=(ADMIN, OWNER))
 
         # Ensure that a member does not upgrade a member to OWNER from ADMIN unless they are an OWNER themselves
-        if self._model.membership_type == ADMIN and updated_fields['membership_type'] == OWNER:
+        if self._model.membership_type.code == ADMIN and updated_fields['membership_type'] == OWNER:
             check_auth(org_id=self._model.org_id, token_info=token_info, one_of_roles=(OWNER))
 
+        # No one can change an OWNER's status, only option is OWNER to leave the team. #2319
```

Comment:
Are we saying OWNER's cannot downgrade themselves any more? As far as I know they should still be able to downgrade their membership if there is another OWNER on the team.

Reply:
OWNER can still downgrade which is change in role (membership type). Owner's status cannot be changed (Active to Inactive) by another owner.

Reply:
Ah got it, I mixed up membership type and status.  Thanks :)